package.json

```diff
@@ -49,7 +51,7 @@
     "generate:locales": "ts-node --skip-project -O '{\"lib\":[\"es2015\"]}' scripts/make-locales.ts",
     "generate:content-definitions": "rimraf definitions/content && mkdir -p definitions/content && gen-api-models --api-spec $npm_package_io_content_specs --out-dir ./definitions/content",
     "generate:bpd": "npm-run-all generate:bpd-citizen generate:bpd-payment generate:bpd-award-periods generate:bpd-winning-transactions",
-    "generate:all": "npm-run-all generate:mock-google-services-json generate:api-definitions generate:pagopa-api-definitions generate:pagopa-api-walletv2 generate:pagopa-cobadge-configuration generate:content-definitions generate:bonus-vacanze-definitions generate:bpd generate:locales",
+    "generate:all": "npm-run-all generate:mock-google-services-json generate:api-definitions generate:pagopa-api-definitions generate:pagopa-api-walletv2 generate:pagopa-cobadge-api generate:pagopa-cobadge-configuration generate:content-definitions generate:bonus-vacanze-definitions generate:bpd generate:cgn-definitions generate:locales",
```

Comment:
```suggestion
    "generate:all": "npm-run-all generate:mock-google-services-json generate:api-definitions generate:pagopa-api-definitions generate:pagopa-api-walletv2 generate:pagopa-cobadge-configuration generate:content-definitions generate:bonus-vacanze-definitions generate:bpd generate:cgn-definitions generate:locales",
```

Reply:
`generate:pagopa-cobadge-api` doesn't exist anymore
I just removed it from `packages.json`
Previously it was there but it doesn't work since it misses `$npm_package_pagopa_cobadge_api`